src/main/java/seedu/address/model/tutee/TuitionTask.java

```diff
@@ -133,6 +133,33 @@ private boolean hasDescription() {
         return !description.equals(NULL_STRING);
     }
 
+    /**
+     * fixes the test but has conflict with Task card
+     * I, a-shakra, didn't write this, but kept it here because i
+     * don't know the reason for it
+    */
+    //@@author a-shakra
+    @Override
+    public boolean equals(Object other) {
```

Comment:
this is redundant with the equal method that I created below. If you look closely, they are exactly the same; it only has different writing format. The conflict with Task card has been removed from some old PRs